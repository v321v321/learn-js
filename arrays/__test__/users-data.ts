export interface User {
  name: string;
  experience: number;
}

export const usersData = [
  { name: 'Zemlak', experience: 14 },
  { name: 'Upton', experience: 13 },
  { name: 'Wisoky', experience: 3 },
  { name: 'Balistreri', experience: 8 },
  { name: 'Braun', experience: 13 },
  { name: 'Walsh', experience: 9 },
  { name: 'Cummings', experience: 3 },
  { name: 'Johnston', experience: 8 },
  { name: 'Dickens', experience: 14 },
  { name: 'Treutel', experience: 6 },
  { name: 'Buckridge', experience: 12 },
  { name: 'Veum', experience: 13 },
  { name: 'Turcotte', experience: 10 },
  { name: 'Bartell', experience: 8 },
  { name: 'Gottlieb', experience: 7 },
  { name: 'Will', experience: 15 },
  { name: 'Gleason', experience: 14 },
  { name: 'Nikolaus', experience: 8 },
  { name: 'Durgan', experience: 4 },
  { name: 'Mayert', experience: 14 },
  { name: 'Flatley', experience: 8 },
  { name: 'Koelpin', experience: 3 },
  { name: 'Feest', experience: 14 },
  { name: 'Thiel', experience: 10 },
  { name: 'Collins', experience: 8 },
  { name: 'Hudson', experience: 6 },
  { name: 'Crona', experience: 7 },
  { name: 'Kihn', experience: 13 },
  { name: 'Murray', experience: 8 },
  { name: 'Rempel', experience: 2 },
  { name: 'Gleichner', experience: 6 },
  { name: 'Keebler', experience: 10 },
  { name: 'Abernathy', experience: 4 },
  { name: 'Wolf', experience: 4 },
  { name: 'Fadel', experience: 11 },
  { name: 'Simonis', experience: 15 },
  { name: 'Ullrich', experience: 7 },
  { name: 'Jones', experience: 5 },
  { name: 'Langworth', experience: 4 },
  { name: 'Lind', experience: 14 },
  { name: 'Kuphal', experience: 9 },
  { name: 'Jerde', experience: 4 },
  { name: 'Fritsch', experience: 4 },
  { name: 'Stanton', experience: 3 },
  { name: 'Turner', experience: 9 },
  { name: 'Nolan', experience: 9 },
  { name: 'Gerlach', experience: 3 },
  { name: 'Boyle', experience: 10 },
  { name: 'Pfannerstill', experience: 6 },
  { name: 'Cormier', experience: 9 },
  { name: 'Padberg', experience: 12 },
  { name: 'Purdy', experience: 13 },
  { name: 'Ritchie', experience: 4 },
  { name: 'Doyle', experience: 12 },
  { name: 'Reichert', experience: 13 },
  { name: 'Breitenberg', experience: 13 },
  { name: 'Erdman', experience: 11 },
  { name: 'Schumm', experience: 9 },
  { name: 'Pfeffer', experience: 3 },
  { name: 'VonRueden', experience: 5 },
  { name: 'Wisozk', experience: 10 },
  { name: 'Metz', experience: 8 },
  { name: 'Barton', experience: 10 },
  { name: 'Ondricka', experience: 9 },
  { name: 'Corkery', experience: 14 },
  { name: 'Okuneva', experience: 13 },
  { name: 'Brown', experience: 13 },
  { name: 'Jacobson', experience: 12 },
  { name: 'Hills', experience: 5 },
  { name: 'Lueilwitz', experience: 4 },
  { name: 'Ziemann', experience: 10 },
  { name: 'Beier', experience: 8 },
  { name: 'Lemke', experience: 5 },
  { name: 'Altenwerth', experience: 2 },
  { name: 'Keeling', experience: 7 },
  { name: 'Crist', experience: 4 },
  { name: 'Deckow', experience: 3 },
  { name: 'Pollich', experience: 6 },
  { name: 'Gusikowski', experience: 7 },
  { name: 'Stehr', experience: 5 },
  { name: 'Jast', experience: 9 },
  { name: 'Nicolas', experience: 4 },
  { name: 'Hermiston', experience: 2 },
  { name: 'Moore', experience: 12 },
  { name: 'Kautzer', experience: 2 },
  { name: 'Huel', experience: 15 },
  { name: 'Heidenreich', experience: 8 },
  { name: 'Conn', experience: 4 },
];
