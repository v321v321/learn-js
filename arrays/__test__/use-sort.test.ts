import { expect, test } from 'vitest';
import {
  sortUsersByExperienceASC,
  sortUsersByExperienceDESC,
  sortUsersByName,
  sortUsersByNameLength,
} from '../examples/use-sort';
import { usersData } from './users-data';

test('Sort users by name', () => {
  const testData = [...usersData];
  sortUsersByName(testData);
  expect(testData).toMatchSnapshot();
});

test('Sort users by experience asc', () => {
  const testData = [...usersData];
  sortUsersByExperienceASC(testData);
  expect(testData).toMatchSnapshot();
});

test('Sort user by experience DESC', () => {
  const testData = [...usersData];
  sortUsersByExperienceDESC(testData);
  expect(testData).toMatchSnapshot();
});

test('Sort user by experience DESC and then by name length ', () => {
  const testData = [...usersData];
  sortUsersByExperienceASC(testData);
  sortUsersByNameLength(testData);
  expect(testData).toMatchSnapshot();
});
