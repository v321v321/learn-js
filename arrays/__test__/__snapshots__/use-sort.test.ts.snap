// Vitest Snapshot v1

exports[`Sort user by experience DESC 1`] = `
[
  {
    "experience": 15,
    "name": "Will",
  },
  {
    "experience": 15,
    "name": "Simonis",
  },
  {
    "experience": 15,
    "name": "Huel",
  },
  {
    "experience": 14,
    "name": "Zemlak",
  },
  {
    "experience": 14,
    "name": "Dickens",
  },
  {
    "experience": 14,
    "name": "Gleason",
  },
  {
    "experience": 14,
    "name": "Mayert",
  },
  {
    "experience": 14,
    "name": "Feest",
  },
  {
    "experience": 14,
    "name": "Lind",
  },
  {
    "experience": 14,
    "name": "Corkery",
  },
  {
    "experience": 13,
    "name": "Upton",
  },
  {
    "experience": 13,
    "name": "Braun",
  },
  {
    "experience": 13,
    "name": "Veum",
  },
  {
    "experience": 13,
    "name": "Kihn",
  },
  {
    "experience": 13,
    "name": "Purdy",
  },
  {
    "experience": 13,
    "name": "Reichert",
  },
  {
    "experience": 13,
    "name": "Breitenberg",
  },
  {
    "experience": 13,
    "name": "Okuneva",
  },
  {
    "experience": 13,
    "name": "Brown",
  },
  {
    "experience": 12,
    "name": "Buckridge",
  },
  {
    "experience": 12,
    "name": "Padberg",
  },
  {
    "experience": 12,
    "name": "Doyle",
  },
  {
    "experience": 12,
    "name": "Jacobson",
  },
  {
    "experience": 12,
    "name": "Moore",
  },
  {
    "experience": 11,
    "name": "Fadel",
  },
  {
    "experience": 11,
    "name": "Erdman",
  },
  {
    "experience": 10,
    "name": "Turcotte",
  },
  {
    "experience": 10,
    "name": "Thiel",
  },
  {
    "experience": 10,
    "name": "Keebler",
  },
  {
    "experience": 10,
    "name": "Boyle",
  },
  {
    "experience": 10,
    "name": "Wisozk",
  },
  {
    "experience": 10,
    "name": "Barton",
  },
  {
    "experience": 10,
    "name": "Ziemann",
  },
  {
    "experience": 9,
    "name": "Walsh",
  },
  {
    "experience": 9,
    "name": "Kuphal",
  },
  {
    "experience": 9,
    "name": "Turner",
  },
  {
    "experience": 9,
    "name": "Nolan",
  },
  {
    "experience": 9,
    "name": "Cormier",
  },
  {
    "experience": 9,
    "name": "Schumm",
  },
  {
    "experience": 9,
    "name": "Ondricka",
  },
  {
    "experience": 9,
    "name": "Jast",
  },
  {
    "experience": 8,
    "name": "Balistreri",
  },
  {
    "experience": 8,
    "name": "Johnston",
  },
  {
    "experience": 8,
    "name": "Bartell",
  },
  {
    "experience": 8,
    "name": "Nikolaus",
  },
  {
    "experience": 8,
    "name": "Flatley",
  },
  {
    "experience": 8,
    "name": "Collins",
  },
  {
    "experience": 8,
    "name": "Murray",
  },
  {
    "experience": 8,
    "name": "Metz",
  },
  {
    "experience": 8,
    "name": "Beier",
  },
  {
    "experience": 8,
    "name": "Heidenreich",
  },
  {
    "experience": 7,
    "name": "Gottlieb",
  },
  {
    "experience": 7,
    "name": "Crona",
  },
  {
    "experience": 7,
    "name": "Ullrich",
  },
  {
    "experience": 7,
    "name": "Keeling",
  },
  {
    "experience": 7,
    "name": "Gusikowski",
  },
  {
    "experience": 6,
    "name": "Treutel",
  },
  {
    "experience": 6,
    "name": "Hudson",
  },
  {
    "experience": 6,
    "name": "Gleichner",
  },
  {
    "experience": 6,
    "name": "Pfannerstill",
  },
  {
    "experience": 6,
    "name": "Pollich",
  },
  {
    "experience": 5,
    "name": "Jones",
  },
  {
    "experience": 5,
    "name": "VonRueden",
  },
  {
    "experience": 5,
    "name": "Hills",
  },
  {
    "experience": 5,
    "name": "Lemke",
  },
  {
    "experience": 5,
    "name": "Stehr",
  },
  {
    "experience": 4,
    "name": "Durgan",
  },
  {
    "experience": 4,
    "name": "Abernathy",
  },
  {
    "experience": 4,
    "name": "Wolf",
  },
  {
    "experience": 4,
    "name": "Langworth",
  },
  {
    "experience": 4,
    "name": "Jerde",
  },
  {
    "experience": 4,
    "name": "Fritsch",
  },
  {
    "experience": 4,
    "name": "Ritchie",
  },
  {
    "experience": 4,
    "name": "Lueilwitz",
  },
  {
    "experience": 4,
    "name": "Crist",
  },
  {
    "experience": 4,
    "name": "Nicolas",
  },
  {
    "experience": 4,
    "name": "Conn",
  },
  {
    "experience": 3,
    "name": "Wisoky",
  },
  {
    "experience": 3,
    "name": "Cummings",
  },
  {
    "experience": 3,
    "name": "Koelpin",
  },
  {
    "experience": 3,
    "name": "Stanton",
  },
  {
    "experience": 3,
    "name": "Gerlach",
  },
  {
    "experience": 3,
    "name": "Pfeffer",
  },
  {
    "experience": 3,
    "name": "Deckow",
  },
  {
    "experience": 2,
    "name": "Rempel",
  },
  {
    "experience": 2,
    "name": "Altenwerth",
  },
  {
    "experience": 2,
    "name": "Hermiston",
  },
  {
    "experience": 2,
    "name": "Kautzer",
  },
]
`;

exports[`Sort user by experience DESC and then by name length  1`] = `
[
  {
    "experience": 4,
    "name": "Conn",
  },
  {
    "experience": 4,
    "name": "Wolf",
  },
  {
    "experience": 8,
    "name": "Metz",
  },
  {
    "experience": 9,
    "name": "Jast",
  },
  {
    "experience": 13,
    "name": "Kihn",
  },
  {
    "experience": 13,
    "name": "Veum",
  },
  {
    "experience": 14,
    "name": "Lind",
  },
  {
    "experience": 15,
    "name": "Huel",
  },
  {
    "experience": 15,
    "name": "Will",
  },
  {
    "experience": 4,
    "name": "Crist",
  },
  {
    "experience": 4,
    "name": "Jerde",
  },
  {
    "experience": 5,
    "name": "Stehr",
  },
  {
    "experience": 5,
    "name": "Lemke",
  },
  {
    "experience": 5,
    "name": "Hills",
  },
  {
    "experience": 5,
    "name": "Jones",
  },
  {
    "experience": 7,
    "name": "Crona",
  },
  {
    "experience": 8,
    "name": "Beier",
  },
  {
    "experience": 9,
    "name": "Nolan",
  },
  {
    "experience": 9,
    "name": "Walsh",
  },
  {
    "experience": 10,
    "name": "Boyle",
  },
  {
    "experience": 10,
    "name": "Thiel",
  },
  {
    "experience": 11,
    "name": "Fadel",
  },
  {
    "experience": 12,
    "name": "Moore",
  },
  {
    "experience": 12,
    "name": "Doyle",
  },
  {
    "experience": 13,
    "name": "Brown",
  },
  {
    "experience": 13,
    "name": "Purdy",
  },
  {
    "experience": 13,
    "name": "Braun",
  },
  {
    "experience": 13,
    "name": "Upton",
  },
  {
    "experience": 14,
    "name": "Feest",
  },
  {
    "experience": 2,
    "name": "Rempel",
  },
  {
    "experience": 3,
    "name": "Deckow",
  },
  {
    "experience": 3,
    "name": "Wisoky",
  },
  {
    "experience": 4,
    "name": "Durgan",
  },
  {
    "experience": 6,
    "name": "Hudson",
  },
  {
    "experience": 8,
    "name": "Murray",
  },
  {
    "experience": 9,
    "name": "Schumm",
  },
  {
    "experience": 9,
    "name": "Turner",
  },
  {
    "experience": 9,
    "name": "Kuphal",
  },
  {
    "experience": 10,
    "name": "Barton",
  },
  {
    "experience": 10,
    "name": "Wisozk",
  },
  {
    "experience": 11,
    "name": "Erdman",
  },
  {
    "experience": 14,
    "name": "Mayert",
  },
  {
    "experience": 14,
    "name": "Zemlak",
  },
  {
    "experience": 2,
    "name": "Kautzer",
  },
  {
    "experience": 3,
    "name": "Pfeffer",
  },
  {
    "experience": 3,
    "name": "Gerlach",
  },
  {
    "experience": 3,
    "name": "Stanton",
  },
  {
    "experience": 3,
    "name": "Koelpin",
  },
  {
    "experience": 4,
    "name": "Nicolas",
  },
  {
    "experience": 4,
    "name": "Ritchie",
  },
  {
    "experience": 4,
    "name": "Fritsch",
  },
  {
    "experience": 6,
    "name": "Pollich",
  },
  {
    "experience": 6,
    "name": "Treutel",
  },
  {
    "experience": 7,
    "name": "Keeling",
  },
  {
    "experience": 7,
    "name": "Ullrich",
  },
  {
    "experience": 8,
    "name": "Collins",
  },
  {
    "experience": 8,
    "name": "Flatley",
  },
  {
    "experience": 8,
    "name": "Bartell",
  },
  {
    "experience": 9,
    "name": "Cormier",
  },
  {
    "experience": 10,
    "name": "Ziemann",
  },
  {
    "experience": 10,
    "name": "Keebler",
  },
  {
    "experience": 12,
    "name": "Padberg",
  },
  {
    "experience": 13,
    "name": "Okuneva",
  },
  {
    "experience": 14,
    "name": "Corkery",
  },
  {
    "experience": 14,
    "name": "Gleason",
  },
  {
    "experience": 14,
    "name": "Dickens",
  },
  {
    "experience": 15,
    "name": "Simonis",
  },
  {
    "experience": 3,
    "name": "Cummings",
  },
  {
    "experience": 7,
    "name": "Gottlieb",
  },
  {
    "experience": 8,
    "name": "Nikolaus",
  },
  {
    "experience": 8,
    "name": "Johnston",
  },
  {
    "experience": 9,
    "name": "Ondricka",
  },
  {
    "experience": 10,
    "name": "Turcotte",
  },
  {
    "experience": 12,
    "name": "Jacobson",
  },
  {
    "experience": 13,
    "name": "Reichert",
  },
  {
    "experience": 2,
    "name": "Hermiston",
  },
  {
    "experience": 4,
    "name": "Lueilwitz",
  },
  {
    "experience": 4,
    "name": "Langworth",
  },
  {
    "experience": 4,
    "name": "Abernathy",
  },
  {
    "experience": 5,
    "name": "VonRueden",
  },
  {
    "experience": 6,
    "name": "Gleichner",
  },
  {
    "experience": 12,
    "name": "Buckridge",
  },
  {
    "experience": 2,
    "name": "Altenwerth",
  },
  {
    "experience": 7,
    "name": "Gusikowski",
  },
  {
    "experience": 8,
    "name": "Balistreri",
  },
  {
    "experience": 8,
    "name": "Heidenreich",
  },
  {
    "experience": 13,
    "name": "Breitenberg",
  },
  {
    "experience": 6,
    "name": "Pfannerstill",
  },
]
`;

exports[`Sort users by experience asc 1`] = `
[
  {
    "experience": 2,
    "name": "Kautzer",
  },
  {
    "experience": 2,
    "name": "Hermiston",
  },
  {
    "experience": 2,
    "name": "Altenwerth",
  },
  {
    "experience": 2,
    "name": "Rempel",
  },
  {
    "experience": 3,
    "name": "Deckow",
  },
  {
    "experience": 3,
    "name": "Pfeffer",
  },
  {
    "experience": 3,
    "name": "Gerlach",
  },
  {
    "experience": 3,
    "name": "Stanton",
  },
  {
    "experience": 3,
    "name": "Koelpin",
  },
  {
    "experience": 3,
    "name": "Cummings",
  },
  {
    "experience": 3,
    "name": "Wisoky",
  },
  {
    "experience": 4,
    "name": "Conn",
  },
  {
    "experience": 4,
    "name": "Nicolas",
  },
  {
    "experience": 4,
    "name": "Crist",
  },
  {
    "experience": 4,
    "name": "Lueilwitz",
  },
  {
    "experience": 4,
    "name": "Ritchie",
  },
  {
    "experience": 4,
    "name": "Fritsch",
  },
  {
    "experience": 4,
    "name": "Jerde",
  },
  {
    "experience": 4,
    "name": "Langworth",
  },
  {
    "experience": 4,
    "name": "Wolf",
  },
  {
    "experience": 4,
    "name": "Abernathy",
  },
  {
    "experience": 4,
    "name": "Durgan",
  },
  {
    "experience": 5,
    "name": "Stehr",
  },
  {
    "experience": 5,
    "name": "Lemke",
  },
  {
    "experience": 5,
    "name": "Hills",
  },
  {
    "experience": 5,
    "name": "VonRueden",
  },
  {
    "experience": 5,
    "name": "Jones",
  },
  {
    "experience": 6,
    "name": "Pollich",
  },
  {
    "experience": 6,
    "name": "Pfannerstill",
  },
  {
    "experience": 6,
    "name": "Gleichner",
  },
  {
    "experience": 6,
    "name": "Hudson",
  },
  {
    "experience": 6,
    "name": "Treutel",
  },
  {
    "experience": 7,
    "name": "Gusikowski",
  },
  {
    "experience": 7,
    "name": "Keeling",
  },
  {
    "experience": 7,
    "name": "Ullrich",
  },
  {
    "experience": 7,
    "name": "Crona",
  },
  {
    "experience": 7,
    "name": "Gottlieb",
  },
  {
    "experience": 8,
    "name": "Heidenreich",
  },
  {
    "experience": 8,
    "name": "Beier",
  },
  {
    "experience": 8,
    "name": "Metz",
  },
  {
    "experience": 8,
    "name": "Murray",
  },
  {
    "experience": 8,
    "name": "Collins",
  },
  {
    "experience": 8,
    "name": "Flatley",
  },
  {
    "experience": 8,
    "name": "Nikolaus",
  },
  {
    "experience": 8,
    "name": "Bartell",
  },
  {
    "experience": 8,
    "name": "Johnston",
  },
  {
    "experience": 8,
    "name": "Balistreri",
  },
  {
    "experience": 9,
    "name": "Jast",
  },
  {
    "experience": 9,
    "name": "Ondricka",
  },
  {
    "experience": 9,
    "name": "Schumm",
  },
  {
    "experience": 9,
    "name": "Cormier",
  },
  {
    "experience": 9,
    "name": "Nolan",
  },
  {
    "experience": 9,
    "name": "Turner",
  },
  {
    "experience": 9,
    "name": "Kuphal",
  },
  {
    "experience": 9,
    "name": "Walsh",
  },
  {
    "experience": 10,
    "name": "Ziemann",
  },
  {
    "experience": 10,
    "name": "Barton",
  },
  {
    "experience": 10,
    "name": "Wisozk",
  },
  {
    "experience": 10,
    "name": "Boyle",
  },
  {
    "experience": 10,
    "name": "Keebler",
  },
  {
    "experience": 10,
    "name": "Thiel",
  },
  {
    "experience": 10,
    "name": "Turcotte",
  },
  {
    "experience": 11,
    "name": "Erdman",
  },
  {
    "experience": 11,
    "name": "Fadel",
  },
  {
    "experience": 12,
    "name": "Moore",
  },
  {
    "experience": 12,
    "name": "Jacobson",
  },
  {
    "experience": 12,
    "name": "Doyle",
  },
  {
    "experience": 12,
    "name": "Padberg",
  },
  {
    "experience": 12,
    "name": "Buckridge",
  },
  {
    "experience": 13,
    "name": "Brown",
  },
  {
    "experience": 13,
    "name": "Okuneva",
  },
  {
    "experience": 13,
    "name": "Breitenberg",
  },
  {
    "experience": 13,
    "name": "Reichert",
  },
  {
    "experience": 13,
    "name": "Purdy",
  },
  {
    "experience": 13,
    "name": "Kihn",
  },
  {
    "experience": 13,
    "name": "Veum",
  },
  {
    "experience": 13,
    "name": "Braun",
  },
  {
    "experience": 13,
    "name": "Upton",
  },
  {
    "experience": 14,
    "name": "Corkery",
  },
  {
    "experience": 14,
    "name": "Lind",
  },
  {
    "experience": 14,
    "name": "Feest",
  },
  {
    "experience": 14,
    "name": "Mayert",
  },
  {
    "experience": 14,
    "name": "Gleason",
  },
  {
    "experience": 14,
    "name": "Dickens",
  },
  {
    "experience": 14,
    "name": "Zemlak",
  },
  {
    "experience": 15,
    "name": "Huel",
  },
  {
    "experience": 15,
    "name": "Simonis",
  },
  {
    "experience": 15,
    "name": "Will",
  },
]
`;

exports[`Sort users by name 1`] = `
[
  {
    "experience": 4,
    "name": "Abernathy",
  },
  {
    "experience": 2,
    "name": "Altenwerth",
  },
  {
    "experience": 8,
    "name": "Balistreri",
  },
  {
    "experience": 8,
    "name": "Bartell",
  },
  {
    "experience": 10,
    "name": "Barton",
  },
  {
    "experience": 8,
    "name": "Beier",
  },
  {
    "experience": 10,
    "name": "Boyle",
  },
  {
    "experience": 13,
    "name": "Braun",
  },
  {
    "experience": 13,
    "name": "Breitenberg",
  },
  {
    "experience": 13,
    "name": "Brown",
  },
  {
    "experience": 12,
    "name": "Buckridge",
  },
  {
    "experience": 8,
    "name": "Collins",
  },
  {
    "experience": 4,
    "name": "Conn",
  },
  {
    "experience": 14,
    "name": "Corkery",
  },
  {
    "experience": 9,
    "name": "Cormier",
  },
  {
    "experience": 4,
    "name": "Crist",
  },
  {
    "experience": 7,
    "name": "Crona",
  },
  {
    "experience": 3,
    "name": "Cummings",
  },
  {
    "experience": 3,
    "name": "Deckow",
  },
  {
    "experience": 14,
    "name": "Dickens",
  },
  {
    "experience": 12,
    "name": "Doyle",
  },
  {
    "experience": 4,
    "name": "Durgan",
  },
  {
    "experience": 11,
    "name": "Erdman",
  },
  {
    "experience": 11,
    "name": "Fadel",
  },
  {
    "experience": 14,
    "name": "Feest",
  },
  {
    "experience": 8,
    "name": "Flatley",
  },
  {
    "experience": 4,
    "name": "Fritsch",
  },
  {
    "experience": 3,
    "name": "Gerlach",
  },
  {
    "experience": 14,
    "name": "Gleason",
  },
  {
    "experience": 6,
    "name": "Gleichner",
  },
  {
    "experience": 7,
    "name": "Gottlieb",
  },
  {
    "experience": 7,
    "name": "Gusikowski",
  },
  {
    "experience": 8,
    "name": "Heidenreich",
  },
  {
    "experience": 2,
    "name": "Hermiston",
  },
  {
    "experience": 5,
    "name": "Hills",
  },
  {
    "experience": 6,
    "name": "Hudson",
  },
  {
    "experience": 15,
    "name": "Huel",
  },
  {
    "experience": 12,
    "name": "Jacobson",
  },
  {
    "experience": 9,
    "name": "Jast",
  },
  {
    "experience": 4,
    "name": "Jerde",
  },
  {
    "experience": 8,
    "name": "Johnston",
  },
  {
    "experience": 5,
    "name": "Jones",
  },
  {
    "experience": 2,
    "name": "Kautzer",
  },
  {
    "experience": 10,
    "name": "Keebler",
  },
  {
    "experience": 7,
    "name": "Keeling",
  },
  {
    "experience": 13,
    "name": "Kihn",
  },
  {
    "experience": 3,
    "name": "Koelpin",
  },
  {
    "experience": 9,
    "name": "Kuphal",
  },
  {
    "experience": 4,
    "name": "Langworth",
  },
  {
    "experience": 5,
    "name": "Lemke",
  },
  {
    "experience": 14,
    "name": "Lind",
  },
  {
    "experience": 4,
    "name": "Lueilwitz",
  },
  {
    "experience": 14,
    "name": "Mayert",
  },
  {
    "experience": 8,
    "name": "Metz",
  },
  {
    "experience": 12,
    "name": "Moore",
  },
  {
    "experience": 8,
    "name": "Murray",
  },
  {
    "experience": 4,
    "name": "Nicolas",
  },
  {
    "experience": 8,
    "name": "Nikolaus",
  },
  {
    "experience": 9,
    "name": "Nolan",
  },
  {
    "experience": 13,
    "name": "Okuneva",
  },
  {
    "experience": 9,
    "name": "Ondricka",
  },
  {
    "experience": 12,
    "name": "Padberg",
  },
  {
    "experience": 6,
    "name": "Pfannerstill",
  },
  {
    "experience": 3,
    "name": "Pfeffer",
  },
  {
    "experience": 6,
    "name": "Pollich",
  },
  {
    "experience": 13,
    "name": "Purdy",
  },
  {
    "experience": 13,
    "name": "Reichert",
  },
  {
    "experience": 2,
    "name": "Rempel",
  },
  {
    "experience": 4,
    "name": "Ritchie",
  },
  {
    "experience": 9,
    "name": "Schumm",
  },
  {
    "experience": 15,
    "name": "Simonis",
  },
  {
    "experience": 3,
    "name": "Stanton",
  },
  {
    "experience": 5,
    "name": "Stehr",
  },
  {
    "experience": 10,
    "name": "Thiel",
  },
  {
    "experience": 6,
    "name": "Treutel",
  },
  {
    "experience": 10,
    "name": "Turcotte",
  },
  {
    "experience": 9,
    "name": "Turner",
  },
  {
    "experience": 7,
    "name": "Ullrich",
  },
  {
    "experience": 13,
    "name": "Upton",
  },
  {
    "experience": 13,
    "name": "Veum",
  },
  {
    "experience": 5,
    "name": "VonRueden",
  },
  {
    "experience": 9,
    "name": "Walsh",
  },
  {
    "experience": 15,
    "name": "Will",
  },
  {
    "experience": 3,
    "name": "Wisoky",
  },
  {
    "experience": 10,
    "name": "Wisozk",
  },
  {
    "experience": 4,
    "name": "Wolf",
  },
  {
    "experience": 14,
    "name": "Zemlak",
  },
  {
    "experience": 10,
    "name": "Ziemann",
  },
]
`;
